changelogs/fragments/lxd-instance-not-found-avoid-false-positives.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "lxd connection plugin - tighten the detection logic for lxd ``Instance not found`` errors, to avoid false detection on unrelated errors such as ``/usr/bin/python3: not found``."
```

Comment:
```suggestion
  - "lxd connection plugin - tighten the detection logic for lxd ``Instance not found`` errors, to avoid false detection on unrelated errors such as ``/usr/bin/python3: not found`` (https://github.com/ansible-collections/community.general/pull/7521)."
```